evdns.c

```diff
@@ -3513,8 +3556,11 @@ evdns_base_set_option_impl(struct evdns_base *base,
 		    val);
 		memcpy(&base->global_nameserver_probe_initial_timeout, &tv,
 		    sizeof(tv));
-	}
-	return 0;
+    } else if (str_matches_option(option, "max-record-len:")) {
```

Comment:
should be enumerated in dns.h (and also feel free to wrap each available option on new line)